OmgvaPOS/src/ItemManagement/Mappers/ItemMappers.cs

```diff
@@ -35,17 +35,17 @@ public static ItemDTO ToItemDTO(this Item item) {
             };
         }
 
-        public static Item ToItem(this ItemDTO itemDTO, Item baseItem) {
+        public static Item FromUpdateRequestToItem(this ItemDTO updateItemDTO, Item baseItem) {
```

Comment:
The old name was good and sufficient, this one is needlessly long as you generally know very well of what kind of object you use the extension method of.

Reply:
In this case the object is generic `ItemDTO` instead of some explicit update request, so the argument "generally know very well of what kind of object you use" is not quite accurate. Tried to create an explicit UpdateItemRequest object, however it introduced a breaking change in some service which required more refactoring. Decided to just rename this method for clarity. Will rename it back because I too don't like this being so long, however it reduces clearity of the method